posthog/api/element.py

```diff
@@ -47,7 +47,8 @@ class ElementViewSet(StructuredViewSetMixin, viewsets.ModelViewSet):
     def stats(self, request: request.Request, **kwargs) -> response.Response:
         filter = Filter(request=request, team=self.team)
 
-        date_from, date_to, date_params = parse_timestamps(filter, team=self.team)
+        _, date_to, date_params = parse_timestamps(filter, team=self.team)
+        date_from = date_from_clause("toStartOfDay", True)
```

Comment:
There's too much magic happening behind those two lines for me to be able to confidently approve them. It's the first time I see this code, and I'm not sure of all the places it can be called. However, just following the thread, how do we know in `date_from_clause` that `params['date_from']` is always a thing? I fear we might get unexpected sentry errors...

Reply:
https://posthog.slack.com/archives/C034XD440RK/p1652969964266189?thread_ts=1652969182.958299&cid=C034XD440RK

We can blame @neilkakkar if it does go wrong 🤣

Reply:
Yep, it's only called from toolbar heatmap. The default case is that it isn't sent - in which case it defaults to seven days ago